contracts/FlowIDTableStaking.cdc

```diff
@@ -1020,8 +1016,10 @@ pub contract FlowIDTableStaking {
                 FlowIDTableStaking.account.save(currentRoleNodeCounts, to: /storage/flowStakingRoleNodeCounts)
             }
 
+            var movesPendingList = FlowIDTableStaking.account.borrow<&{String: {UInt32: Bool}}>(from: /storage/idTableMovesPendingList)
+                ?? panic("No moves pending list in account storage")
```

Comment:
Maybe make this a function and call it in the other places where this borrow occurs. This way the type/path is in one place, the code becomes less brittle

Reply:
I only do it twice, so creating a whole other function seems kind of unnecessary

Reply:
Saw three occurrences, but also works as-is (there a few other cases for other paths, too). Keeping it DRY is in general a good idea to avoid future problems